dapr/actor/runtime/reminderdata.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+
+"""
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT License.
+"""
+
+import base64
+
+from datetime import timedelta
+from typing import Any, Dict
+
+
+class ActorReminderData:
+    """represents Actor reminder data."""
+    def __init__(
+            self, name: str, state: bytes,
+            due_time: timedelta, period: timedelta):
+        self._name = name
+        self._due_time = due_time
+        self._period = period
+
+        if not isinstance(state, (str, bytes)):
+            raise ValueError(f'only str and bytes are allowed for state: {type(state)}')
+
+        if isinstance(state, str):
+            self._state = state.encode('utf-8')
+        else:
+            self._state = state
+
+    @property
+    def name(self) -> str:
+        return self._name
+
+    @property
+    def state(self) -> bytes:
+        return self._state
+
+    @property
+    def due_time(self) -> timedelta:
+        return self._due_time
+
+    @property
+    def period(self) -> timedelta:
+        return self._period
+
+    def as_dict(self) -> dict:
+        encoded_state = None
+        if self._state is not None:
+            encoded_state = base64.b64encode(self._state)
+        return {
+            'name': self._name,
+            'dueTime': self._due_time,
+            'period': self._due_time,
```

Comment:
does this get serialized into the period format as required by dapr runtime  1h5m5s?

Reply:
yes json serializer converts to golang format.  please see the code and tests:

https://github.com/dapr/python-sdk/blob/6f07dd55a529c7782bc9703ce28c3da63fa83f3d/dapr/serializers/json.py#L66-L67

https://github.com/dapr/python-sdk/blob/youngp/add-timer/tests/serializers/testutil.py